models/reviews.models.js

```diff
@@ -0,0 +1,24 @@
+const db = require("../db/connection");
+
+exports.selectReview = (id) => {
+  if (/\D/g.test(id) === true) {
+    return Promise.reject({
+      status: 400,
+      message: "Invalid id provided, a review id must be a number.",
+    });
```

Comment:
SQL will actually throw and error already if the incorrect data type is inserted into a column so we dont have to do this check, we can instead just allow db.query to run it and then catch and act on the error accordingly